module/trace/trace.go

```diff
@@ -236,14 +225,28 @@ func (t *Tracer) StartSpanFromParent(
 	operationName SpanName,
 	opts ...trace.SpanStartOption,
 ) trace.Span {
-	if !IsSampled(parentSpan) {
```

Comment:
I finally figured out the real regression.  Removing this caused the tracer to create a real span, but it doesn't show up on profile.

Reply:
on the plus side, execution just got 1-2% faster cuz of the txn id pre-computations